.github/CONTRIBUTING.rst

```diff
@@ -67,6 +67,7 @@ Here's how to make a one-off code change.
       $ git checkout -b your-branch-name
 
 3. **Make a commit to your feature branch**. Each commit should be self-contained and have a descriptive commit message that helps other developers understand why the changes were made.
+   We also have a check-list for PRs below.
```

Comment:
maybe direct link to it?

Reply:
Are there referencing schemes for that that work both [here](https://docs.python-telegram-bot.org/en/stable/contributing.html) and [here](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/.github/CONTRIBUTING.rst) ? 🤔

Reply:
can't remember one, maybe something to ask chatgpt for

Reply:
don't have an account xD

Reply:
so you've never.. used it?

Anyway here's what it said, which I verified at least works in RTD:

```suggestion

   We also have a check-list for PRs `below`_.
   
End of file:
.. _`below`: #check-list-for-prs

```